### DISCUSSION TITLE
Theme Park Support?

### DISCUSSION THREAD

**Discussion by iamlite** [hassio-addons/addon-nginx-proxy-manager#455] [General]
https://docs.theme-park.dev/themes/nginx-proxy-manager/

Would love to use this with this add-on. 

![hotpink](https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/119023701/4ce8e6f3-b26f-4f68-8bc2-0d332d32c8e0)
![aquamarine](https://github.com/hassio-addons/addon-nginx-proxy-manager/assets/119023701/2880dbfb-53bc-482e-8479-1e08e2073998)

**Comment by frenck** [comment]
I have no intention of adding this.

While cool, it is a mod that needs maintenance. In the end, this is a control panel that one is not in every day. I don't think the juice it worth the squeeze for that penalty.

Sorry!

../Frenck
